ui/js/components/auth.js

```diff
@@ -161,6 +162,14 @@ treeherder.component("loginCallback", {
                 key: $location.search().accessToken,
                 algorithm: 'sha256'
             };
+
+            // Cribbed from taskcluster-tools. Save this for interacting with tc
+            const results = $location.search();
+            if (typeof results.certificate === 'string') {
```

Comment:
I think we can use `certificate` from above rather than needing to use $location again? :-)